### DISCUSSION TITLE
Include a Yesterday page

### DISCUSSION THREAD

**Discussion by gand07** [bellrichm/weewx-jas#123] [Q&A] [answered]
I want to include the Yesterday page, so in weewx.cont under [[[[pages]]]], I added:
[[[[[yesterday]]]]]
		enable = True
		[[[[[[minmax]]]]]]
                [[[[[[outTemp]]]]]]
                [[[[[[outHumidity]]]]]]
                [[[[[[barometer]]]]]]
                [[[[[[rain]]]]]]
                [[[[[[wind]]]]]]
                [[[[[[ET]]]]]]
                [[[[[[UV]]]]]]
                [[[[[[radiation]]]]]]

Yesterday is displayed fine in navbar, but when I click on Yesterday the page is blank. So I'm sure I've forgotten something

Gert

**Comment by gand07** [comment, marked as answer]
After a couple of hours, it was  working. Still learning

**Reply by bellrichm** [reply]
Interesting. Yesterday data is only generated once a day (since it only changes once). But when I add yesterday, it generates on the first run it was added.  I’ll keep an eye on this. I might even just generate it every archive period. With some of the recent performance improvements it should be doable.
Thanks for the feedback.
